airflow/providers/google/cloud/utils/credentials_provider.py

```diff
@@ -89,6 +89,8 @@ def provide_gcp_credentials(key_file_path: Optional[str] = None, key_file_dict:
     :type key_file_path: str
     :param key_file_dict: Dictionary with credentials.
     :type key_file_dict: Dict
+
+    __ https://cloud.google.com/docs/authentication/production
```

Comment:
TIL -- didn't knew we could do this in docstrings too

Reply:
Yup, Doc strings are "just" rst :)